WikilogUtils.php

```diff
@@ -328,7 +328,7 @@ public static function categoryList( $list ) {
 		foreach ( $list as $cat ) {
 			$title = Title::makeTitle( NS_CATEGORY, $cat );
 			$categoryUrl = $title->getPrefixedText();
-			$categoryTxt = $title->getText();
+			$categoryTxt = $title->getContent()->getNativeData();
```

Comment:
same as above.

Reply:
Sorry my bad, I think `$title` here is not related with `WikiPage`